packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -1960,60 +1969,142 @@ describe('Checkout', () => {
                     }
                 });
 
+                const payload = {
+                    action: 'submit'
+                };
+
                 // Act
                 await wrapper.vm.onFormSubmit();
 
                 // Assert
-                expect(wrapper.emitted(EventNames.CheckoutValidationError).length).toBe(1);
-                expect(wrapper.emitted(EventNames.CheckoutValidationError)[0][0]).toEqual(mockValidationState);
+                expect(trackFormInteractionSpy).toHaveBeenCalledWith(payload);
             });
 
-            it('should try to call `submitCheckout` if form is Valid', async () => {
-                // Arrange
-                isFormValidSpy.mockReturnValue(true);
+            describe('when the form is Invalid', () => {
+                let wrapper;
+                let getFormValidationStateSpy;
 
-                const wrapper = mount(VueCheckout, {
-                    store: createStore(),
-                    i18n,
-                    localVue,
-                    propsData,
-                    mocks: {
-                        $v,
-                        $logger
-                    }
+                const mockValidationState = {
+                    validFields: [
+                        'customer.mobileNumber',
+                        'address.line1',
+                        'address.city',
+                        'address.postcode'
+                    ],
+                    invalidFields: []
+                };
+
+                beforeEach(() => {
+                    isFormValidSpy.mockReturnValue(false);
+
+                    getFormValidationStateSpy = jest.spyOn(validations, 'getFormValidationState');
+                    getFormValidationStateSpy.mockReturnValue(mockValidationState);
+
+                    wrapper = mount(VueCheckout, {
+                        store: createStore(),
+                        i18n,
+                        localVue,
+                        propsData,
+                        mocks: {
+                            $v,
+                            $logger
+                        }
+                    });
                 });
 
-                // Act
-                await wrapper.vm.onFormSubmit();
+                it('should emit `CheckoutValidationError`', async () => {
+                    // Act
+                    await wrapper.vm.onFormSubmit();
 
-                // Assert
-                expect(submitCheckoutSpy).toHaveBeenCalled();
+                    // Assert
+                    expect(wrapper.emitted(EventNames.CheckoutValidationError).length).toBe(1);
+                });
+
+                it('should call `trackFormInteraction` with and action `inline_error` and validation errors', async () => {
```

Comment:
Nit: should "...with and action..." read "...with an action..."?

Reply:
🤦‍♀️